app/_locales/es/messages.json

```diff
@@ -931,7 +931,7 @@
     "message": "Tu firma ya fue solicitada"
   },
   "contractInteraction": {
-    "message": "Interacción del Contrato"
+    "message": "Interacción con contrato"
```

Comment:
should we maintain capital letter on the word `contrato`?

Reply:
It's usual in English but is not correct in Spanish. It's true that is being more common in Spanish because of English influence.